packages/collector/test/tracing/misc/specification_compliance/tracer_compliance_test_cases.json

```diff
@@ -730,8 +730,48 @@
     "X-INSTANA-T out": "fa2375d711a4ca0f",
     "X-INSTANA-S out": "$new_span_id_2",
     "X-INSTANA-L out": "1",
-    "traceparent out": "00-0000000000000000fa2375d711a4ca0f-$new_span_id_2-01",
+    "traceparent out": "00-0000000000000000fa2375d711a4ca0f-$new_span_id_2-03",
     "tracestate out": "in=fa2375d711a4ca0f;$new_span_id_2",
     "index": 40
+  },
+  {
+    "Scenario": "INSTANA_SERVICE_NAME support / entry span",
+    "What to do?": "add span.data.service on entry span",
+    "X-INSTANA-T in": "fa2375d711a4ca0f",
+    "X-INSTANA-S in": "37cb2d6e9b1c078a",
+    "Server-Timing": "intid;desc=fa2375d711a4ca0f",
+    "entrySpan.t": "fa2375d711a4ca0f",
+    "entrySpan.p": "37cb2d6e9b1c078a",
+    "entrySpan.s": "$new_span_id_1",
+    "entrySpan.data.service": "Tracer Test Suite Service",
+    "exitSpan.t": "fa2375d711a4ca0f",
+    "exitSpan.p": "$new_span_id_1",
+    "exitSpan.s": "$new_span_id_2",
+    "X-INSTANA-T out": "fa2375d711a4ca0f",
+    "X-INSTANA-S out": "$new_span_id_2",
+    "X-INSTANA-L out": "1",
+    "traceparent out": "00-0000000000000000fa2375d711a4ca0f-$new_span_id_2-03",
+    "tracestate out": "in=fa2375d711a4ca0f;$new_span_id_2",
+    "index": 41
+  },
+  {
+    "Scenario": "INSTANA_SERVICE_NAME support / exit span",
+    "What to do?": "add span.data.service on exit span",
+    "X-INSTANA-T in": "fa2375d711a4ca0f",
+    "X-INSTANA-S in": "37cb2d6e9b1c078a",
+    "Server-Timing": "intid;desc=fa2375d711a4ca0f",
+    "entrySpan.t": "fa2375d711a4ca0f",
+    "entrySpan.p": "37cb2d6e9b1c078a",
+    "entrySpan.s": "$new_span_id_1",
+    "exitSpan.t": "fa2375d711a4ca0f",
+    "exitSpan.p": "$new_span_id_1",
+    "exitSpan.s": "$new_span_id_2",
+    "exitSpan.data.service": "Tracer Test Suite Service",
+    "X-INSTANA-T out": "fa2375d711a4ca0f",
+    "X-INSTANA-S out": "$new_span_id_2",
+    "X-INSTANA-L out": "1",
+    "traceparent out": "00-0000000000000000fa2375d711a4ca0f-$new_span_id_2-03",
+    "tracestate out": "in=fa2375d711a4ca0f;$new_span_id_2",
+    "index": 42
```

Comment:
This two new test cases are unrelated to the change at hand, they are the result of copying over a fresh copy of the tracer test suite test case definition.